app/models/user.rb

```diff
@@ -122,6 +122,16 @@ def self.with_2sv_statuses(scope_names)
     relation
   end
 
+  def self.with_default_permissions
```

Comment:
I think this method can be simplified as `new(supported_permissions: SupportedPermission.default)`.

Reply:
Fixed in [cbf2077](https://github.com/alphagov/signon/pull/2442/commits/cbf207774211c8897f2bbef972ffceb5619d2fb1)